### PR TITLE
gvls: init at 0.14.3

### LINKED ISSUE #83118: Package gvls
gnome-builder now needs gvls for vala support: https://gitlab.gnome.org/esodan/gvls

### PR DESCRIPTION
**Pull request by ethancedwards8**
###### Motivation for this change
Means to replace: https://github.com/NixOS/nixpkgs/pull/83386
Also closes https://github.com/NixOS/nixpkgs/issues/83118

Very small changes, just made it more up to date.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/gvls/default.nix [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    repo = "gvls";
+    rev = "${pname}-${version}";
+    sha256 = "1brz3dxdjfww1b8npzrpzanndmy835w6fvkfydjbqz1h89myv73a";
+  };
+
+  nativeBuildInputs = [
+    gobject-introspection
+    meson
+    ninja
+    pkgconfig
```
Comment (SuperSandro2000) [inline pkgs/development/tools/gvls/default.nix:37]:
```suggestion
    pkg-config
```

File: pkgs/development/tools/gvls/default.nix [hunk 2]
```diff
@@ -0,0 +1,63 @@
+{ stdenv
+, lib
+, fetchFromGitLab
+, meson
+, ninja
+, pkgconfig
```
Comment (SuperSandro2000) [inline pkgs/development/tools/gvls/default.nix:6]:
```suggestion
, pkg-config
```

File: pkgs/development/tools/gvls/default.nix [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+  propagatedBuildInputs = [
+    glib
+    gtk3
+    gtksourceview3
+    json-glib
+    jsonrpc-glib
+    libgee
+    vala
+  ];
```
Comment (SuperSandro2000) [inline pkgs/development/tools/gvls/default.nix:54]: Do we really want to propagate all of those?

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/development/tools/gvls/default.nix (hunk 3 above)
Comment (ethancedwards8) [reply pkgs/development/tools/gvls/default.nix:54]: Not sure, as I'm inheriting the codebase from worldofpeace. I've only built it and tested the binary. I'll look into it more and come to a conclusion.

**Pull request closed without merging**